microsoft/onefuzz
# fix deleting repro VMs

## Summary of the Pull Request

As is, repro VMs are not properly deleted due to a logic bug.

## PR Checklist

## Info on Pull Request

Fixes the logic when deleting repro VMs.

## Validation Steps Performed

Create a repro, note the VM ID.  Then delete it.  Wait ~5 minutes.  Check the Azure portal.  Resources related to $VM_ID
